core/templates/pages/exploration-editor-page/translation-tab/translation-tab.component.ts

```diff
@@ -133,16 +132,13 @@ angular.module('oppia').component('translationTab', {
       };
 
       $scope.showWelcomeTranslationModal = function() {
-        $uibModal.open({
-          template: require(
-            'pages/exploration-editor-page/translation-tab/' +
-            'modal-templates/welcome-translation-modal.template.html'),
+        NgbModal.open(WelcomeTranslationModalComponent, {
           backdrop: true,
-          controller: 'WelcomeTranslationModalController',
           windowClass: 'oppia-welcome-modal'
         }).result.then(function(explorationId) {
           SiteAnalyticsService.registerAcceptTutorialModalEvent(
             explorationId);
+          $rootScope.$applyAsync();
```

Comment:
Is this line needed?

Reply:
Yes, without this line the ` translation tutorial doesn't start `